lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -0,0 +1,60 @@
+{
+  "workflows": [
+    {
+      "name": "medidate_depsoit",
+      "label": "Self-deposit workflow",
+      "description": "A multi-step workflow for self-depoist",
+      "actions": [
+         {
+           "name": "finalize_digitization",
+           "from_states": [{"names": ["pending"], "roles": ["reviewing"]}],
+           "transition_to": "metadata_review",
+           "attributes": {
+             "presentation_sequence": 1
+           }
+         }, {
+           "name": "finalize_metadata",
+           "from_states": [{"names": ["metadata_review"], "roles": ["finalizing"]}],
+           "transition_to": "final_review",
+           "attributes": {
+                "presentation_sequence": 2
+           }
+         }, {
+           "name": "complete",
+           "from_states": [{"names": ["final_review"], "roles": ["completing"]}],
+           "transition_to": "complete",
+           "attributes": {
+             "presentation_sequence": 3
+           },
+           "notifications": [
+             {
+               "notification_type": "email",
+               "name": "Sufia::Workflow::ConfirmSubmission",
```

Comment:
This line will cause a failure unless `Sufia::Workflow::ConfirmSubmission` is a real thing. Remove notifications block from JSON.